test/backend-unit-test/run_test.sh

```diff
@@ -53,8 +53,11 @@ TEST_DIR=backend/src
 
 echo "Clone ML pipeline code in COMMIT SHA ${COMMIT_SHA}..."
 git clone https://github.com/${GITHUB_REPO} ${BASE_DIR}
-cd ${BASE_DIR}/${TEST_DIR}
-git checkout ${COMMIT_SHA}
+cd ${BASE_DIR}
```

Comment:
The unit test image are not built as part of pipeline. please run the make file and update the image tag too. thanks

Reply:
Thanks for telling me. I've updated the unit-test image.